wordle-web/pages/index.vue

```diff
@@ -41,17 +41,17 @@
 import { Game, GameState } from "../scripts/game";
 import Axios from "axios";
 
-const game: Ref<Game> = ref(new Game(""));
-// Get random word from word list
-getWordFromApi().then((word) => {
-  game.value = new Game(word);
-});
-
+const game: Ref<Game> = ref(new Game("GAMES"));
 provide("GAME", game.value);
 
 const myGuess = ref("");
 
 onMounted(() => {
+  // Get random word from word list
+  getWordFromApi().then((word) => {
+    game.value = new Game(word);
+  });
```

Comment:
Be sure to add a leaderboard page leaderboard.vue once you get the implementation squared away with the controller!